resources/frameworks_2021Q3.yaml

```diff
@@ -0,0 +1,91 @@
+---
+
+#########################
+### AutoML frameworks ###
+#########################
+
+AutoGluon:
+  version: "0.3.1"
+
+AutoGluon_benchmark:
+  extends: AutoGluon
+  description: "AutoGluon with 'best_quality' preset provides the most accurate overall predictor"
+  params:
+    presets: best_quality
+
+autosklearn:
+  version: '0.13.0'
+
+autosklearn2:
+  extends: autosklearn
+  params:
+    _askl2: true
+
+autoxgboost:
+  version: '#908631665f4c763d548ed203d48f05fe68613844'
+
+AutoWEKA:
+  version: '2.6'
+
+flaml:
+  version: '0.6.0'
+
+GAMA:
+  version: '21.0.0'
+
+H2OAutoML:
+  version: '3.34.0.1'
+
+lightautoml:
+  version: '0.2.16'
+
+mljarsupervised:
+  version: '0.10.6'
+
+mljarsupervised_benchmark:
+  extends: mljarsupervised
+  description: "MLJar using 'Compete' mode to provide most accurate predictor"
+  params:
+    mode: Compete
+
+MLNET:
+  version: '16.5.26'
+
+MLPlan:
+  version: '0.2.4'
+
```

Comment:
removed the `bastract` here? didn't it work?

Reply:
I'm somewhat sure the PR originated from before abstract definitions were used.
Sorry for not double checking before asking the review. Will update this and others.